src/components/Sidebar/LatestComments.js

```diff
@@ -28,7 +28,13 @@ const LatestComments = ({ comments }) => (
     <div className="LatestComments__container">
       <h4 className="LatestComments__title"><i className="iconfont icon-time LatestComments__icon" /> Latest Comments</h4>
       <div className="LatestComments__divider" />
-      {comments && comments.map(comment => renderComment(comment))}
+      {comments && comments.map(comment => (<RenderComment
+        key={comment.id}
+        text={comment.text}
+        author={comment.author}
+        created={comment.created}
+      />
+      ))}
```

Comment:
Intend look not good. These 2 should not be in same level:
```
/>
))}
```

You can do like this: 
```
{comments && comments.map(comment => (
  <RenderComment key={comment.id} {...comment} />
))}
```